src/docs/dependencies.dot

```diff
@@ -1,11 +1,16 @@
 digraph dependencies {
-  quest -> {slam primal mint spin};
-  {quest slam primal mint spin} -> {slic core};
-  mint -> sidre [style="dashed"];
-  spin -> {slam primal};
-  sidre -> {slic core};
-  slic -> core;
-  slic -> lumberjack [style="dashed"];
-  lumberjack -> core;
-  inlet -> {sidre slic primal core};
+  compound=true;
+  style="rounded";
+  subgraph cluster_basic {
+    slic -> core;
+    slic -> lumberjack [style="dashed"];
+    lumberjack -> core;
+  }
+  subgraph cluster_app {
+    quest -> {slam primal mint spin};
+    mint -> sidre [style="dashed"];
+    spin -> {slam primal};
+    inlet -> {sidre primal};
+  }
+  primal -> slic [ltail=cluster_app, lhead=cluster_basic];
 }
```

Comment:
I'm not a fan of this reorganization of the dependencies.
I'm not sure that I agree with the characterization of "basic" vs. "app" and find the new graph harder to read.

I'd recommend reverting this change and we can discuss this change as a group.

See the following screenshot:
![image](https://user-images.githubusercontent.com/5626552/132905270-b0aa6380-5b73-4181-b2aa-51e29948526a.png)

Reply:
The most noticeable thing about the previous graph was that every component has a dependency on slic and core.  This added a lot of noise to the graph.

I'll revert the graph.  Yes, it will be a good thing to discuss how best to show dependencies.

Reply:
Reverted.  @kennyweiss , please let me know what you think.

Reply:
I agree. I prefer the way it was before.